pom.xml

```diff
@@ -56,13 +54,46 @@
                 </execution>
             </executions>
         </plugin>
+           <plugin>
+               <groupId>org.apache.maven.plugins</groupId>
+               <artifactId>maven-release-plugin</artifactId>
+               <version>2.5.1</version>
+           </plugin>
       </plugins>
     </build>
 
     <groupId>com.qubole.presto</groupId>
     <artifactId>udfs</artifactId>
-    <name>udfs</name>
-    <version>0.1.0-SNAPSHOT</version>
+    <version>0.1.3-SNAPSHOT</version>
```

Comment:
shall we revert version to 0.1.0-SNAPSHOT? or have we already deployed to nexus staging so need to keep this version?